### PR TITLE
remove na option added

### PR DESCRIPTION
**Pull request by microsud**

### CODE REVIEW

File: R/summaries.R [hunk 1]
```diff
@@ -13,7 +13,10 @@
 #'   median or prevalence. Default is 'mean'.
 #'
 #' @param abund_values A \code{character} value to select an
-#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assayNames}}
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assayNames}} 
+#'   
+#' @param remove_na For \code{getTopTaxa} logical argument for calculation method
```
Comment (antagomir) [inline R/summaries.R]: I suggest to use the argument name `na.rm` which is used in many different functions and packages. It would be more standard.

**Review by antagomir — APPROVED** [review]

File: R/summaries.R (hunk 1 above)
Comment (microsud) [reply R/summaries.R]: OK will make the change 😀

**Comment by antagomir** [comment]
There is  build error, can you check and fix before we can merge? 

edit: I am working on it now.

**Comment by antagomir** [comment]
The build error is now fixed in #202 , waiting to be merged to master. It's a few new lines in .github/workflows

**Comment by antagomir** [comment]
Hmm.. the fix in #202 has not been merged to master yet.. seems to me that this was merged without the checks passing.

**Comment by antagomir** [comment]
We should open this back after #202 has been accpepted..

**Pull request merged**